101-webapp-linux-django/azuredeploy.json

```diff
@@ -0,0 +1,91 @@
+{
+    "contentVersion": "1.0.0.0",
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
+            "defaultValue": "[concat('Django-', uniqueString(resourceGroup().name, utcNow('F')))]",
```

Comment:
make this idempotent - use resourceGroup().id and remove utcnow()

Reply:
Updated